platform-dom/src/main/java/com/softicar/platform/dom/event/IDomEvent.java

```diff
@@ -118,6 +118,33 @@ public interface IDomEvent {
 	 */
 	DomRect getBoundingClientRect();
 
+	/**
+	 * Returns the delta-x value of a {@link DomEventType#WHEEL} event.
+	 * <p>
+	 * Returns 0 for any other {@link DomEventType}.
+	 *
+	 * @return the delta-x value (never <i>null</i>)
+	 */
+	Double getDeltaX();
```

Comment:
why support nullability when we don't need it?
```suggestion
	double getDeltaX();
```

Reply:
sure that